ui/apps/platform/src/Containers/ConfigManagement/Dashboard/widgets/UsersWithMostClusterAdminRoles.js

```diff
@@ -11,10 +11,10 @@ import Lollipop from 'Components/visuals/Lollipop';
 import ReactRouterPropTypes from 'react-router-prop-types';
 
 const QUERY = gql`
-    query usersWithClusterAdminRoles {
+    query usersWithClusterAdminRoles($query: String) {
         clusters {
```

Comment:
Why are we querying clusters?

Reply:
@md2119 
We loop through each cluster, and then the subjects within each cluster, to build a map of users with `clusterAdmin` permission, and how many.

The code for that is further down in this file,
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/ConfigManagement/Dashboard/widgets/UsersWithMostClusterAdminRoles.js#L32-L55

This was written by Assaf over 3 years ago; if there is now a more efficient way to get those counts to display in the widget, let the UI know.

Reply:
It seems not needed but let's leave the code as is because it could be brittle.